syntax = "proto3";

package fraud_detection;

service FraudDetectionService {
    rpc FraudDetection (FraudDetectionRequest) returns (FraudDetectionResponse);
}

message FraudDetectionRequest {
    string card_number = 1;
    string order_amount = 2;
}

message FraudDetectionResponse {
    bool is_fraud = 1;
}
